### DISCUSSION TITLE
如何从多个目录读取pkg

### DISCUSSION THREAD

**Discussion by constly** [ejoy/ant#103] [General]
有些pkg会多个项目共用， 比如像unity里面的dotween这种
以及如果要支持mod，那可能也会存在从其他目录读取数据情况

目前引擎支持从多个目录读取pkg吗~  如何读呢

**Comment by cloudwu** [comment]
vfs 的工作机制是按优先级合并多个本地目录：https://github.com/ejoy/ant/wiki/VFS

目前是把 engine 和项目本地目录以及 mod 目录合并起来，你也可以在 .mount 中指定更多。

参考这里的代码：https://github.com/ejoy/ant/blob/master/engine/mount.lua#L7-L25

**Reply by constly** [reply]
可以了，感谢！
